src/state/slices/opportunitiesSlice/selectors/lpSelectors.ts

```diff
@@ -237,7 +261,7 @@ export const selectAggregatedEarnUserLpOpportunities = createDeepEqualOutputSele
           bnOrZero(aggregatedLpAssetBalance)
             .times(opportunityMetadata?.underlyingAssetRatiosBaseUnit[i] ?? '0')
```

Comment:
similarly this should exist

Reply:
72393201feb8e9577409935017201ff87b231cbf